WcaOnRails/spec/features/incident_management_spec.rb

```diff
@@ -31,7 +31,7 @@
 
       scenario "filters by text" do
         visit "/incidents"
-        within(:css, ".search") do
+        within(:css, ".incidents-log-table-container") do
```

Comment:
For the record I had to change this selector because `search` was also used by the dropdown, and it makes sense to scope it with something related to the incidents log anyway.